scribble-lib/scribble/blueboxes.rkt

```diff
@@ -44,14 +65,19 @@
         (class/interface-tag->constructor-tag
          (definition-tag->class/interface-tag tag))
         cache))
-     (append plain-strs
-             (if constructor-strs '("") '())
-             (if constructor-strs (cdr constructor-strs) '()))]
+     (assert
```

Comment:
I think we should fix this by just improving the type for `append` in Typed Racket.